projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.scss

```diff
@@ -52,13 +47,30 @@
             height: 2.75rem;
           }
 
-          .carousel-text {
+          .carousel-text-container {
+            display: flex;
+            flex-direction: column;
+            height: 2rem;
+            justify-content: center;
             text-align: center;
-            font-size: 0.625rem;
-            max-width: 3.5rem;
-            line-height: .8rem;
+
+            .carousel-text {
+              display: flex;
+              justify-content: center;
+              align-items: center;
+              font-size: 0.625rem;
+              max-width: 3.5rem;
+              line-height: .8rem;
+              min-height: 1.5rem;
+            }
+  
+            .organ-number {
```

Comment:
Is there a better class name/way to describe extra text below the carousel item?

Reply:
Would donor-registrations work?

Reply:
I was thinking something more generic like num-results